configure.ac

```diff
@@ -29,6 +29,71 @@ AC_TYPE_SIZE_T
 AC_TYPE_UINT32_T
 AC_TYPE_UINT8_T
 
+# Checks for required binaries
+AC_PATH_PROG([SHELL], [sh], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$SHELL" == x"no" ; then
+   SHELL="/bin/sh"
+   AC_MSG_NOTICE([using $SHELL as SHELL])
+fi
+AC_SUBST(SHELL)
+AC_DEFINE_UNQUOTED([SHELL_PATH], ["$SHELL"], [Path to sh binary])
+
+AC_PATH_PROG([TAR], [tar], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$TAR" == x"no" ; then
+   TAR="/bin/tar"
+   AC_MSG_NOTICE([using $TAR as TAR])
+fi
+AC_SUBST(TAR)
+AC_DEFINE_UNQUOTED([TAR_PATH], ["$TAR"], [Path to tar binary])
+
+AC_PATH_PROG([HWCLOCK], [hwclock], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$HWCLOCK" == x"no" ; then
+   HWCLOCK="/bin/hwclock"
+   AC_MSG_NOTICE([using $HWCLOCK as HWCLOCK])
+fi
+AC_SUBST(HWCLOCK)
+AC_DEFINE_UNQUOTED([HWCLOCK_PATH], ["$HWCLOCK"], [Path to hwclock binary])
+
+AC_PATH_PROG([MODPROBE], [modprobe], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$MODPROBE" == x"no" ; then
+   MODPROBE="/sbin/modprobe"
+   AC_MSG_NOTICE([using $MODPROBE as MODPROBE])
+fi
+AC_SUBST(MODPROBE)
+AC_DEFINE_UNQUOTED([MODPROBE_PATH], ["$MODPROBE"], [Path to modprobe binary])
+
+AC_PATH_PROG([DEPMOD], [depmod], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$DEPMOD" == x"no" ; then
+   DEPMOD="/sbin/depmod"
+   AC_MSG_NOTICE([using $DEPMOD as DEPMOD])
+fi
+AC_SUBST(DEPMOD)
+AC_DEFINE_UNQUOTED([DEPMOD_PATH], ["$DEPMOD"], [Path to depmod binary])
+
+AC_PATH_PROG([IPTABLES], [iptables], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$IPTABLES" == x"no" ; then
+   IPTABLES="/sbin/iptables"
+   AC_MSG_NOTICE([using $IPTABLES as IPTABLES])
+fi
+AC_SUBST(IPTABLES)
+AC_DEFINE_UNQUOTED([IPTABLES_PATH], ["$IPTABLES"], [Path to iptables binary])
+
+AC_PATH_PROG([IPTABLES_RESTORE], [iptables-restore], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$IPTABLES_RESTORE" == x"no" ; then
+   IPTABLES_RESTORE="/sbin/iptables-restore"
+   AC_MSG_NOTICE([using $IPTABLES_RESTORE as IPTABLES_RESTORE])
+fi
+AC_SUBST(IPTABLES_RESTORE)
+AC_DEFINE_UNQUOTED([IPTABLES_RESTORE_PATH], ["$IPTABLES_RESTORE"], [Path to iptables-restore binary])
+
+AC_PATH_PROG([IPTABLES_SAVE], [iptables-save], [no], [$PATH:/sbin:/usr/sbin])
+if test x"$IPTABLES_SAVE" == x"no" ; then
+   IPTABLES_SAVE="/sbin/iptables-save"
+   AC_MSG_NOTICE([using $IPTABLES_SAVE as IPTABLES_SAVE])
+fi
+AC_SUBST(IPTABLES_SAVE)
+AC_DEFINE_UNQUOTED([IPTABLES_SAVE_PATH], ["$IPTABLES_SAVE"], [Path to iptables-save binary])
+
```

Comment:
could you remove all the change in this file and directly define BUSYBOX_BINARIES in the build/make-initrd.sh